static/js/ui_util.ts

```diff
@@ -48,3 +48,9 @@ export function update_unread_count_in_dom(unread_count_elem: JQuery, count: num
     unread_count_span.show();
     unread_count_span.text(count);
 }
+
+export function parse_html(html: string): DocumentFragment {
+    const template = document.createElement("template");
+    template.innerHTML = html;
+    return template.content;
+}
```

Comment:
Can you add a docstring/comment documenting this function?

Reply:
The type signature seems like fine documentation?